app/views/components/autocompleteCountryList.scala.html

```diff
@@ -59,8 +59,7 @@ <h1 class="govuk-label-wrapper">
     text = v,
     selected = k == form("country").value.getOrElse(""),
     attributes = Map("id" → s"country-$k"))
-    },
-    formGroupClasses = "form-field-group"
```

Comment:
Please revert and update this, thanks. This should be updated to `formGroup = FormGroup(classes = Some("form-field-group"))` instead as per [9.4.0](https://github.com/hmrc/play-frontend-hmrc/blob/main/CHANGELOG.md#940---2024-03-27). You will need to import `FormGroup` i.e. `@import uk.gov.hmrc.govukfrontend.views.viewmodels.FormGroup`